### DISCUSSION TITLE
What's the equivalent of libxml2 XML_PARSE_RECOVER ? How to ignore errors while parsing ?

### DISCUSSION THREAD

**Discussion by Altyrost** [zeux/pugixml#593] [General]
What's the equivalent of libxml2 XML_PARSE_RECOVER ? How to ignore errors while parsing ?

**Comment by zeux** [comment]
I'm not fully familiar with how libxml2 implements recovery mode, but pugixml doesn't support recovery and expects a valid XML document as an input. If a document is invalid, pugixml will return *both* an error status and an incomplete tree after parsing, but it doesn't have a way to "continue" parsing after the error is encountered.

**Comment by Altyrost** [comment]
Thank you.
